hadoop/pom.xml

```diff
@@ -49,6 +49,61 @@
     </dependencies>
     <build>
         <plugins>
+            <!--
+        Generate an OSGI compatible MANIFEST file.
+      -->
+            <plugin>
```

Comment:
I think the tsfile and service-rpc pom files are modified because the two modules are dependent by jdbc.
But why this Hadoop pom is also modified?